impress-org/form-builder-library
# refactor: update OptionsPanel: Hide Minus Icon When Only One Option & Add showToggle property

## Resolves: 
These changes are mentioned in the Multiple Tribute Type design file https://app.zeplin.io/project/640922426926771dd88e528a/screen/6696d8840aaa2408a00c2fab


## Description
This PR introduces two key updates:

### Removal of Minus Icon for Single Options: 
The minus icon is now hidden when there is only one option available. This change prevents users from completely removing all options from the panel, thereby ensuring that at least one option remains.

### Addition of `showToggle` Property: 
A new showToggle property has been added to the main component. This property provides users with the ability to hide the value toggle, offering more control over the component's visibility and behavior.

## Affects
OptionsPanel component

## Visuals
![Screen Recording 2024-08-06 at 12 06 04 PM](https://github.com/user-attachments/assets/7589db26-8329-46c7-9472-b4dd8cd337a8)


## Testing Instructions
- Install and run locally
- Use component while adding showToggle prop & ensure it can hide the value toggle.
- Remove all options until one remains & ensure it does not allow you to remove the last option.
## Pre-review Checklist
